cicecore/cicedynB/infrastructure/comm/mpi/ice_boundary.F90

```diff
@@ -2384,6 +2385,69 @@ subroutine ice_HaloUpdate2DI4(array, halo,                    &
 
  end subroutine ice_HaloUpdate2DI4
 
+!***********************************************************************
+
+ subroutine ice_HaloUpdate2DL1(array, halo,                    &
```

Comment:
I think I'd name it `ice_HaloUpdate2DL` since there's no concepts of "precision" for logicals, so we do not add any information by having this trailing `1`.

Reply:
It's a good point and I thought about that when I implemented it.  I guess I like that all the subroutine names are as similar as possible, even if it doesn't make sense always.  But happy to change this if there is consensus.  This interface name is not directly used, so it's not really critical either way.

Reply:
I'm ambivalent.  The 1 is fine...

Reply:
Lets leave it as is.